Misc/NEWS.d/next/Library/2023-11-29-04-43-32.gh-issue-105323.zelrW9.rst

```diff
@@ -0,0 +1 @@
+Remove ``WITH_APPLE_EDITLINE`` to use the same declaration for all ``editline``
```

Comment:
No end user impact. We can skip NEWS.d

Reply:
Sure, do you want me to remove this file and label the PR?

Reply:
> Sure, do you want me to remove this file and label the PR?

I already added the label, just please remove the file :)